promscale/installation/source.md

```diff
@@ -0,0 +1,80 @@
+# Promscale from source
+You can install Promscale from source on any operating system. The Promscale
+source files are available from our
+[GitHub releases page][gh-promscale-download].
+
+Before you begin, you must have an already installed and working Prometheus
+environment. Additionally, you need a
+[self-hosted TimescaleDB instance][tsdb-install-self-hosted] installed.
```

Comment:
Aha, in reference to my above comment about the extension: this is probably the bit where it's "hidden". As far as I know, none of the installation mechanisms on the referred page have the `promscale_extension` packaged.